ufobalapp/static/ufobalapp/ng-parts/tournament_match.html

```diff
@@ -38,18 +38,6 @@
         </span>
     </div>
 
-    <div class="row">
-        <div class="columns small-6 medium-5 text-right clickable" ng-class="match.team1.color" ng-click="newGoalieChange(match.team1)">
-            <b ng-show="match.team1.goalie">{{ match.team1.goalie.nickname }}</b>
-            <span ng-hide="match.team1.goalie">bez brankáře</span>
-        </div>
-        <div class="columns medium-2 show-for-medium-up text-center">&nbsp;</div>
-        <div class="columns small-6 medium-5 clickable" ng-class="match.team2.color" ng-click="newGoalieChange(match.team2)">
-            <b ng-show="match.team2.goalie">{{ match.team2.goalie.nickname }}</b>
-            <span ng-hide="match.team2.goalie">bez brankáře</span>
-        </div>
-    </div>
-
```

Comment:
Proč se tohle maže? Celkem to tam chybí

Reply:
huh musel byt missclick a pak jsem to prehlidl